db/migrations/20170127143150_create_sign_in_credentials.cr

```diff
@@ -0,0 +1,13 @@
+class CreateSignInCredentials::V20170127143150 < LuckyMigrator::Migration::V1
+  def migrate
+    create :sign_in_credentials do
+      add user_id : Int32
+      add email : String
+      add encrypted_password : String
```

Comment:
Usually when writing test it is best to only add what is necessary to verify everything works. So in this case you could remove email and encrypted_password